vowpalwabbit/example.h

```diff
@@ -60,6 +60,43 @@ typedef union
   VW::continuous_actions::probability_density_function_value pdf_value;  // probability density value for a given action
 } polyprediction;
 
+class gd_prediction_cache
```

Comment:
where is this `get_prediction_cache` used? I can't find it being used anywhere

Reply:
This was an experiment.  It has been removed.

Reply:
I see what happened.  Did not show up as a change in my vscode.  Fixed.